setup.py

```diff
@@ -39,14 +39,18 @@ def run_tests(self):
         'scripts/manage-views.py'
     ],
     install_requires=[
-        'SQLAlchemy', 'geoalchemy2', 'psycopg2', 'alembic', 'black'
+        "geoalchemy2==0.6.3",
```

Comment:
From what I understood from our [Zulip conversation](https://pcic.zulipchat.com/#narrow/stream/183656-CSG-general/topic/setup.2Epy.20vs.20requirements.2Etxt/near/188151629), we were going to move toward open ended specifiers in `setup.py` and specific dependencies in `requirements.txt`. Did I misunderstand? Should these dependencies not be "abstract dependencies" rather than concrete?

Reply:
Correct, my bad.